README.md

```diff
@@ -65,11 +69,53 @@ processors:
   ...
 ```
 
-## Installation
+## Installing requirements
 
 SDP is officially supported for Python 3.8, but might work for other versions.
 
-SDP depends on NeMo toolkit (ASR and nemo-text-processing parts).
-Please follow (NeMo installation instructions)[https://github.com/NVIDIA/NeMo#installation].
-After that, run `pip install -r requirements.txt` and (optionally)
-`pip install -r tests/requirements.txt` if you want to run tests.
\ No newline at end of file
+SDP depends on NeMo toolkit (ASR, NLP, and nemo-text-processing parts).
+Please follow [NeMo installation instructions](https://github.com/NVIDIA/NeMo#installation).
+
+After that, run `pip install -r requirements.txt` and (optionally) `pip install -r tests/requirements.txt` if you want to run tests.
+
+## Running SDP
+After installing the requirements for SDP, to use it, you will need to download it using e.g. `git clone https://github.com/NVIDIA/NeMo-speech-data-processor.git`.
+
+### Run config in repository
+To run processing for a dataset that has a config and all processor code inside the SDP directory structure, run commands like below:
+
+```bash
+python <SDP_ROOT>/main.py \
+    --config-path="dataset_configs/spanish_pc/mcv12/" \
+    --config-name="config.yaml" \
+    data_split="test" \
+    workspace_dir="<dir where processed data will be saved, and where initial data tar file is already located>"
+```
+
+### Run own config
+To run processing for a dataset with a config that is not inside the SDP directory structure, but all processors are within the SDP directory, run commands like:
+
+```bash
+python <SDP_ROOT>/main.py \
+    --config-path="<path to config: either absolute path or relative path *from SDP_ROOT directory*>" \
+    --config-name="<config file name>.yaml" \
+    data_split="test" \ # or other data_split
+    workspace_dir="<dir to use as workspace>"
+```
+
+### Run own config and use own SDP processors
+To run processing for a dataset with a config that is not inside the SDP directory structure, and at least one processor is outside the SDP directory structure run:
+
+```bash
+PYTHONPATH=<path to dir containing your custom processor either directly or in subdirectory> python <SDP_ROOT>/main.py \
```

Comment:
Is this really necessary? I think it worked for me without python path env

Reply:
I would expect it to work without specifying PYTHONPATH if the new file is downstream of <SDP_ROOT> but not otherwise

Reply:
Maybe add a quick comment about this, so that people know that if they just copy paste existing config in the new folder inside the repo, they don't have to always provide that variable